.gitlab-ci.yml

```diff
@@ -10,8 +10,8 @@ variables:
   MENDER_ADDON_CONNECT_VERSION: 1.0.0-build2
   # Make sure to update the link in mender-docs to the new one when changing
   # this.
-  RASPBIAN_URL: https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2021-01-12/2021-01-11-raspios-buster-armhf-lite.zip
-  RASPBIAN_NAME: 2021-01-11-raspios-buster-armhf-lite
+  RASPBIAN_URL: http://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-05-28/2020-05-27-raspios-buster-lite-armhf.zip
```

Comment:
Is this a typo?
```suggestion
  RASPBIAN_URL: https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-05-28/2020-05-27-raspios-buster-lite-armhf.zip
```

Reply:
Good catch.

The original comment upgraded http->https, and here I just reverted the full commit without paying attention to that detail. We'll fix this in master and then cherry-pick back